integration/pom.xml

```diff
@@ -20,6 +20,7 @@
         <hawtbufVersion>1.1</hawtbufVersion>
         <activemqProtobuf>1.1</activemqProtobuf>
         <groovy.version>2.4.21</groovy.version>
+        <jsonSchemaValidatorVersion>2.2.14</jsonSchemaValidatorVersion>
```

Comment:
This is only the test project. You should change this dependency in the non test project.